code/game/objects/objs.dm

```diff
@@ -34,6 +34,7 @@
 	var/suicidal_hands = FALSE // Does it requires you to hold it to commit suicide with it?
 	/// Is it emagged or not?
 	var/emagged = FALSE
+	var/cmagged = FALSE
```

Comment:
make this a trait. I do not want a var on every single obj to track if its emagged by a clown.

Reply:
Done.